python/binary_search.py

```diff
@@ -0,0 +1,22 @@
+def binarysearch(sp,ep,l1,item):
+    for i in range(sp,ep):
+        mid=(sp+ep)//2
+        if l1[mid]==item:
+            return mid
+        elif item>l1[mid]:
+            sp=mid+1
+        elif item<l1[mid]:
+            ep=mid-1
+    else:
+        return 0
+l=eval((input()))

```

Comment:
Codacy found an issue: [Use of possibly insecure function - consider using safer ast.literal_eval.](https://app.codacy.com/gh/tasnimzotder/hacktoberfest-2020/pullRequest?prid=6279655)